spec/takeaway_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:order_klass) {
+    double :order_klass, new: menu_klass, calculate_quantities: order,
+    calculate_cost: order, calculate: order
+  }
+  subject(:takeaway) { described_class.new(order_klass) }
+  let(:order) {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)